e2e/jest.test-setup.ts

```diff
@@ -152,7 +157,7 @@ const getPageTitle = async () => {
       return title.textContent;
     })
     .catch(() => {
-      return 'getPageTitle() func failed';
```

Comment:
I was little confused by this string in log. Remove it to avoid confusion in future.